### PR TITLE
Update epp to version 3.1.0

### PR DESCRIPTION
**Pull request by jurriaan**
Can someone with a mac generate the bottles as described in the readme?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,10 @@ On Linux, install [Linuxbrew](http://linuxbrew.sh) first.
 * upload the generated `*.tar.gz` file to Google Cloud Storage
 
   ```
+  # If you used --force-core-tap, you need to rename the generated tar so it uses '-' instead of '--'.
+  # Otherwise brew downloading the bottle will not find the correct file.
+  # e.g.: epp--3.1.0.big_sur.bottle.tar.gz to epp-3.1.0.big_sur.bottle.tar.gz
+
```
Comment (oscaredel) [inline README.md:55]: This is a bit of a strange bug when you use that option: https://github.com/Homebrew/brew/issues/4740

I noticed the Catalina version also had the double dashes at https://homebrew-blendle.storage.googleapis.com so I assume that option of --force-core-tap is used before and needed?

**Review by oscaredel — APPROVED** [review]

**Comment by Jasperste** [comment]
@oscaredel do you happen to know if there is a way to test this branch locally on my mac as well? It constantly falls back to 3.0.0 when doing different things to install it again

**Comment by oscaredel** [comment]
@Jasperste Do you mean just installing epp?
First one uses instructions from our tap and searches the cloud storage for the binary (where it can't find it because of the double dashes), second one uses instructions from the local file. I guess it will get 3.1.0 from the tap once this is merged?
```
➜  homebrew-blendle git:(epp-update) brew uninstall epp
Uninstalling /usr/local/Cellar/epp/3.1.0... (5 files, 6.7MB)
➜  homebrew-blendle git:(epp-update) brew install epp
==> Installing epp from blendle/blendle
==> Downloading https://homebrew-blendle.storage.googleapis.com/epp-3.0.0.catalina.bottle.tar.gz
#=#=#                                                                         
curl: (22) The requested URL returned error: 404 
Error: Failed to download resource "epp"
Download failed: https://homebrew-blendle.storage.googleapis.com/epp-3.0.0.catalina.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Cloning https://github.com/blendle/epp.git
Updating /Users/oscaredel/Library/Caches/Homebrew/epp--git
==> Checking out tag v3.0.0
Previous HEAD position was b0be242 Merge pull request #12 from blendle/fix-funcs-in-templates
HEAD is now at e3a91c4 Merge pull request #11 from blendle/add_wildcard_support
HEAD is now at e3a91c4 Merge pull request #11 from blendle/add_wildcard_support
==> make build
🍺  /usr/local/Cellar/epp/3.0.0: 5 files, 5.8MB, built in 4 seconds
```

```
➜  homebrew-blendle git:(epp-update) brew uninstall epp
Uninstalling /usr/local/Cellar/epp/3.0.0... (5 files, 5.8MB)
➜  homebrew-blendle git:(epp-update) brew install Formula/epp.rb 
==> Downloading https://homebrew-blendle.storage.googleapis.com/epp-3.1.0.big_sur.bottle.tar.gz
Already downloaded: /Users/oscaredel/Library/Caches/Homebrew/downloads/86eb9679ee893cecdae431d5f3ce167586797c3e2bc43fa75ac43e60d0023857--epp-3.1.0.big_sur.bottle.tar.gz
==> Pouring epp-3.1.0.big_sur.bottle.tar.gz
🍺  /usr/local/Cellar/epp/3.1.0: 5 files, 6.7MB
```

**Comment by Jasperste** [comment]
@oscaredel I have the same when doing that. I don't see any reference to bottle, but have never done something very interesting with homebrew before.
```
~homebrew-blendle [epp-update] $ brew install Formula/epp.rb
Updating Homebrew...
==> Auto-updated Homebrew!
Updated Homebrew from fc305cc9e to c8210ac3c.
Updated 2 taps (homebrew/core and homebrew/cask).
==> New Formulae
efm-langserver                                                           libpinyin
==> Updated Formulae
Updated 761 formulae.
==> Updated Casks
Updated 147 casks.

==> Cloning https://github.com/blendle/epp.git
Updating /Users/jasperste/Library/Caches/Homebrew/epp--git
From https://github.com/blendle/epp
 * [new tag]         v3.1.0     -> v3.1.0
==> Checking out tag v3.1.0
Previous HEAD position was e3a91c4 Merge pull request #11 from blendle/add_wildcard_support
HEAD is now at b0be242 Merge pull request #12 from blendle/fix-funcs-in-templates
HEAD is now at b0be242 Merge pull request #12 from blendle/fix-funcs-in-templates

==> make build
🍺  /usr/local/Cellar/epp/3.1.0: 5 files, 6.7MB, built in 6 seconds
```

**Comment by oscaredel** [comment]
@Jasperste What version of macOS are you using? 
Maybe it only gets the bottle if it is the same as the OS in the formula? Otherwise I don't know.
It is possible to mention multiple bottles: https://docs.brew.sh/Bottles

**Comment by Jasperste** [comment]
@oscaredel  10.15.7

**Comment by oscaredel** [comment]
@Jasperste That is Catalina, while the mentioned bottle is Big Sur. Might be the reason.

**Pull request merged**